etc/templates/generic.slurm.template

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -l
+#SBATCH -p $QUEUE
+#SBATCH -N 1
+#SBATCH -t $WALL_CLOCK
+#SBATCH --ntasks-per-node 1
+#SBATCH --cpus-per-task $CPUS
+#SBATCH -J glide_$USER_NAME
+
+MB=$(( 4000*$CPUS ))
```

Comment:
It would be nice to make amount of memory per core a configurable value instead of hard-coding it. Having said that, I don't know how much work it would require so I'll be completely fine with making this improvement in another ticket.

Reply:
In the past, the ctrl_execute + ctrl_platform_*  setup would schedule full nodes,  so glide-ins with partial nodes, chopping them up nicely is a new area, really the new feature of this ticket.  We are starting with a simple approach that aligns with what s3df Slurm does -- if you request a core, it provides 4 GB of memory be default.  So in the current setup, yes, to get  "8 GB/core " for the workflow, one must schedule twice the number of cores (cores will be idle).   It is true that further advances in this package could/should make the memory per core customizable,  but I think it can be added complexity when we are ready for it.    Making those cores available for low memory / core work is nice, but I suspect the project doesn't have a lot of such work.